src/scenegraph/Serializer.h

```diff
@@ -149,7 +149,7 @@ namespace Serializer {
 				throw std::out_of_range("Serializer::Reader encountered truncated stream.");
 #endif
 
-			out = *reinterpret_cast<const T *>(m_at);
```

Comment:
Did you forget `#include <cstring>` ?

Reply:
It's pulled in by other headers but wouldn't hurt to explicitly have declared here as well.